pages/platform/ai/gi_07_manage_registry/guide.en-gb.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Use & manage existing and new image registries
+slug: use-manage-registries
+excerpt: Learn how to use and manage your public and private image registries
+section: General information
+order: 109
+updated: 2023-05-02
+---
+
+**Last updated 2nd May, 2023.**
+
+## Objective
+
+This guide is intended to help you **use and manage your public and private registries** through ovhai command line interface (AI CLI), API and OVHcloud control panel. This will help you when working with **OVHcloud AI Tools**, especially with **AI Training** and **AI Deploy**.
+
+## Requirements
+
+- A **Public Cloud** project
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+
+## Instructions
+
+This documentation is divided into the following parts:
+- Manage existing registries through
+    - OVHcloud Control Panel (UI)
+    - OVHcloud AI CLI
+- Create, configure and use new registries
+    - Private Harbor registry
+    - Private Docker registry
+    - GitHub registry (packages)
+
+## Manage existing registries
+
+The Public Cloud provide a default registry called `shared registry`, where users of a same Public Cloud project are able to push their custom images, and use them through our AI Tools. 
+
+> [!warning]
+>
+>This `shared registry` can help you perform your tests, but **should not be used in production**, as we reserve the right to delete its content if deemed necessary.
+>
+
+This is why it can be interesting to add, manage other registries. We can either do this by using the OVHcloud Control Panel (UI) or the AI CLI.
```

Comment:
```suggestion
This is why it can be interesting to add, manage other registries. We can either do this by using the OVHcloud Control Panel (UI) or the `ovhai` CLI.
```